internal/cli/atlas/deployments/search/indexes/create.go

```diff
@@ -78,7 +81,12 @@ func (opts *CreateOpts) Run(ctx context.Context) error {
 		return err
 	}
 
-	opts.index, err = opts.mongodbClient.Database(opts.index.Database).CreateSearchIndex(ctx, opts.index.CollectionName, opts.index)
+	db := opts.mongodbClient.Database(opts.index.Database)
+	if _, err = db.SearchIndexByName(ctx, opts.index.Name, opts.index.CollectionName); err != mongodbclient.ErrSearchIndexNotFound {
+		return ErrSearchIndexDuplicated
+	}
```

Comment:
should this be something like this?
```suggestion
	if idx, _ = db.SearchIndexByName(ctx, opts.index.Name, opts.index.CollectionName); idx != nil {
		return ErrSearchIndexAlreadyExists
	}
```

Reply:
sure, that works as well